app/models/models.py

```diff
@@ -26,7 +30,7 @@ class Voter(db.Base):
     id = Column(String(50), primary_key=True, index=True, default=uuid4().hex)
     village = Column(String(255), ForeignKey("villages.id"))
     name = Column(String(255), index=True)
-    contact = Column(String(255), ForeignKey("contacts.id"))
+    contact = Column(String(255), ForeignKey("contactus.id"))
```

Comment:
You are certain that the tablename is called `contactus`?, to enable code readability i would suggest using the format `ForeignKey(ContactUs.id)` as opposed to `ForeignKey("contactus.id")`